department-of-veterans-affairs/va.gov-cms
# VACMS-2164: Adding publisher listing access perms.

## Description
See #2164 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/86369172-112e3a00-bc4c-11ea-93f0-794bfb31ae9b.png)

## QA steps
1. As user #34, go to `node/2803/edit` and verify that access is not denied.
2. Edit a couple of fields, and confirm content is changed on successful save.
